internal/servicebus/namespace.go

```diff
@@ -3,32 +3,66 @@ package servicebus
 import (
 	"errors"
 
+	"github.com/Azure/azure-amqp-common-go/v3/auth"
 	servicebussdk "github.com/Azure/azure-service-bus-go"
 
 	"github.com/keikumata/azure-pub-sub/internal/aad"
 )
 
 const (
+	// TODO:take this value from config or environment
 	serviceBusResourceURI = "https://servicebus.azure.net/"
```

Comment:
this works for AzurePublicCloud but will need to be different for other cloud (Mooncake, for example)